release.sh

```diff
@@ -9,5 +9,7 @@ summon docker run --rm -t \
   --env-file @SUMMONENVFILE \
   puppet-pdk \
   bash -ec """
-    pdk release --skip-documentation --skip-changelog --force
+    PDK_DISABLE_ANALYTICS=true pdk release --skip-documentation \
+                                           --skip-changelog \
```

Comment:
why do we skip docs and changelog? we were publishing those with each release before, and it was nice

Reply:
As far as I can tell this only prevents them from being auto-generated/updated by `pdk release` action: 
```
...
--skip-changelog | Skips the automatic Changelog generation.
--skip-documentation | Skips the documentation update.
```

It should still use the static versions of those things from the repo.

Reply:
ok. let's try it with this release, and if the  docs/changelog don't update we'll remove these for the next one